packages/connection/src/class-plugin-storage.php

```diff
@@ -19,6 +19,22 @@
  */
 class Plugin_Storage {
 
+	const ACTIVE_PLUGINS_OPTION_NAME = 'jetpack_connection_active_plugins';
```

Comment:
`ACTIVE_PLUGINS` appears to be a little bit ambiguous. May be use `CONNECTED_PLUGINS` instead, what do you think?

Reply:
active plugins, at least for now without the soft disconnect, seems more precise. we don't know if we are connected or not, we only know there is an active plugin that asks to use the connection

Reply:
Good point.